spec-dtslint/operators/zipAll-spec.ts

```diff
@@ -0,0 +1,25 @@
+import { of } from 'rxjs';
+import { zipAll } from 'rxjs/operators';
+
+it('should infer correctly', () => {
+  const o = of(of(1, 2, 3)).pipe(zipAll()); // $ExpectType Observable<number[]>
+});
+
+it('should support projecting values', () => {
+  const o = of(of(1, 2, 3)).pipe(zipAll(value => String(value))); // $ExpectType Observable<string>
+});
+
+it('should support custom typings on the projector', () => {
+  const o = of(of(1, 2, 3)).pipe(zipAll<string>(value => value)); // $ExpectType Observable<string>
+  const p = of(of(1, 2, 3)).pipe(zipAll<string>((value: string) => value)); // $ExpectType Observable<string>
```

Comment:
If this test passes, it looks to me like there must be an error in the typings - there is no `string` in the source observable.

Reply:
I sure can 😄 !

Reply:
this might actually be a problem with `pipe` typings, not `zipAll`. :thinking:  I think we can just remove these tests for now and file a separate issue.